docs/leaderboard/index.md

```diff
@@ -57,11 +57,11 @@ Check out the [Node.js example](https://github.com/momentohq/client-sdk-javascri
 ## Frequently asked questions about Momento Topics
 
 <details>
-  <summary>What's the difference between Momento Leaderboard and Redis Sorted Sets?</summary>
-  Momento Leaderboard has a much more efficient memory footprint, allowing us to scale to tens of millions of records, and the APIs were built with gaming use cases specifically in mind.
+  <summary>What's the difference between Momento Leaderboards and Redis Sorted Sets?</summary>
+  Momento Leaderboards has a much more efficient memory footprint, allowing us to scale to tens of millions of records, and the APIs were built with gaming use cases specifically in mind.
 </details>
 
 <details>
-  <summary>Do Momento Leaderboard items expire?</summary>
-  No, Momento Leaderboard items have no TTL.
-</details>
\ No newline at end of file
+  <summary>Do Momento Leaderboards items expire?</summary>
+  No, Momento Leaderboards items have no TTL.
```

Comment:
should this answer be updated to say 7 day ttl by default?

Reply:
haha yes probably